### PR TITLE
22. Read & Delete Supplier API

### PR DESCRIPTION
**Pull request by mhdfikri44**

### CODE REVIEW

File: admin/script/supplier_data.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                        `</td>
+                        <td>` +
+                        dataSupplier[i].alamat +
+                        `</td>
+                        <td>` +
+                        dataSupplier[i].hp +
+                        `</td>
+                        <td>
+                            <button id="hapus" class="btn btn-danger" value="` +
+                        dataSupplier[i].id +
```
Comment (aldhi88) [inline admin/script/supplier_data.js]: `dataSupplier[i].id` ganti ke `dataSupplier[i].kode`
@MhdFikri44

File: admin/script/supplier_data.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                        `">
+                            Hapus
+                            </button>
+                            <button id="edit" class="btn btn-primary">
+                                <a class="text-light" href="?page=supplier_edit&id=` +
+                        dataSupplier[i].id +
```
Comment (aldhi88) [inline admin/script/supplier_data.js]: `dataSupplier[i].id` ganti ke `dataSupplier[i].kode`
@MhdFikri44

**Review by aldhi88 — CHANGES_REQUESTED** [review]

File: admin/script/supplier_data.js [hunk 3]
```diff
@@ -31,13 +31,13 @@ $(document).ready(function () {
...
-                        dataSupplier[i].id +
+                        dataSupplier[i].kode +
                         `">
                             Hapus
                             </button>
                             <button id="edit" class="btn btn-primary">
                                 <a class="text-light" href="?page=supplier_edit&id=` +
-                        dataSupplier[i].id +
+                        dataSupplier[i].kode +
                         `">Edit</a>
```
Comment (mhdfikri44) [inline admin/script/supplier_data.js:41]: @aldhi88

File: admin/script/supplier_data.js [hunk 4]
```diff
@@ -31,13 +31,13 @@ $(document).ready(function () {
                         `</td>
                         <td>
                             <button id="hapus" class="btn btn-danger" value="` +
-                        dataSupplier[i].id +
+                        dataSupplier[i].kode +
                         `">
```
Comment (mhdfikri44) [inline admin/script/supplier_data.js:35]: @aldhi88

**Review by aldhi88 — APPROVED** [review]

**Pull request merged**